frontend/templates/views/partials/search.j2

```diff
@@ -76,63 +76,61 @@
                     reset-on-refresh
                     single-item
           >
-            {% raw %}
```

Comment:
I recall you mentioned to me that `{% raw %}` doesn't work here anymore. Can you quickly recap why? Sorry I forgot 😉

Reply:
It still works with `{% raw %}` but it was very ugly and unreadable. I just wanted to clean it a little. 😊